### ISSUE TITLE
Showing GitHub repos that are no longer exist

### ISSUE THREAD

**Issue by yg** [thepracticaldev/dev.to#7751]
**Describe the bug**

Showing GitHub repos that are no longer exist

I added some GitHub repos to the profile a long ago and renamed my GitHub account and deleted that repo too

**To Reproduce**

1. Add some repo to the profile
2. Remove GitHub integration

**Screenshots**

![Screenshot](https://user-images.githubusercontent.com/61746440/81465516-3a9c8d80-91e8-11ea-9d60-60642c134341.png)

**Desktop (please complete the following information):**

1. OS: Ubuntu 19.04
2. Browser: Chrome Beta
3. Version: 83.0.4103.23 (Official Build) beta (64-bit)

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Thanks for raising this issue yg! We're currently working on improving the code around fetching GitHub repos and we'll see if can fix this as well at the same time.

**Comment by rhymes** [comment]
To our awesome contributors, please note that this issue will be worked on by a DEV team member and is not up for grabs.

**Comment by jrg94** [comment]
Yes, I'm noticing this same issue as someone who transferred a couple repos to a GitHub organization. It's still there, but it doesn't seem to update or allow me to remove it.

**Comment by rhymes** [comment]
@yg @jrg94 could you try again? We shipped #8382 which should help!

**Comment by yg** [comment]
@rhymes Still the repos are there even after hard refresh!

**Comment by rhymes** [comment]
@yg in your profile page or in https://dev.to/settings/integrations ?

**Comment by yg** [comment]
@rhymes It is in my profile page

**Comment by rhymes** [comment]
if you go to https://dev.to/settings/integrations it should delete the repositories that are gone and then they should disappear from the profile page as well. It's a bit convoluted but let me know if that works. We still have to make this process automatic

**Comment by jrg94** [comment]
Seems to work! At the very least, it removes the organization repos. Only problem now is the interface doesn't seem to line up with what I actually see on my profile. The backend is displaying a lot more selected pins than I actually see on my profile. 

![image](https://user-images.githubusercontent.com/11050412/84532995-47b00f00-acb5-11ea-9842-e347e8096a18.png)

![image](https://user-images.githubusercontent.com/11050412/84533040-5dbdcf80-acb5-11ea-9b1b-91ab31925d8e.png)


![image](https://user-images.githubusercontent.com/11050412/84533012-50084a00-acb5-11ea-8165-9c041dc8cce2.png)

**Comment by yg** [comment]
@rhymes Now it is fixed! The problem is might be cache!

**Comment by jrg94** [comment]
Yeah, I ran a few of those fun Shift+F5s in Chrome, but nothing was updating on my end. That said, everything looks good on the front end! 👍

**Comment by rhymes** [comment]
Thanks @yg for checking!

@jrg94 we'll take a look at that odd behavior

**Comment by rhymes** [comment]
@jrg94 we merged https://github.com/thepracticaldev/dev.to/pull/8676 which should address the problem. can you go once more to https://dev.to/settings/integrations and then to your profile?

**Comment by jrg94** [comment]
Profile seems to be fine, but the integrations is broken again. Now, it seems to load forever, only to timeout with the following error in red text: "An error occurred: Error". I've attached the log. 

[dev.to-1592404372233.log](https://github.com/thepracticaldev/dev.to/files/4793173/dev.to-1592404372233.log)

**Comment by rhymes** [comment]
@jrg94 thank you and :-(
